src/main/java/com/dreammaster/gthandler/GT_MachineRecipeLoader.java

```diff
@@ -2332,21 +2332,21 @@ public void run3() {
         GT_Values.RA.addAssemblerRecipe(new ItemStack[]{GT_OreDictUnificator.get(OrePrefixes.wireFine, Materials.AnnealedCopper, 4), GT_OreDictUnificator.get(OrePrefixes.dustSmall, Materials.GalliumArsenide, 1), GT_Utility.getIntegratedCircuit(4)}, Materials.Plastic.getMolten(144L), ItemList.Circuit_Parts_Diode.get(6L), 600, 30);
         GT_Values.RA.addAssemblerRecipe(new ItemStack[]{GT_OreDictUnificator.get(OrePrefixes.wireFine, Materials.AnnealedCopper, 4), ItemList.Circuit_Silicon_Wafer.get(1L), GT_Utility.getIntegratedCircuit(4)}, Materials.Plastic.getMolten(144L), ItemList.Circuit_Parts_Diode.get(4L), 600, 30);
         GT_Values.RA.addAssemblerRecipe(new ItemStack[]{GT_OreDictUnificator.get(OrePrefixes.wireFine, Materials.Platinum, 16), GT_OreDictUnificator.get(OrePrefixes.dust, Materials.GalliumArsenide, 1),  GT_Utility.getIntegratedCircuit(3)}, Materials.Plastic.getMolten(576L), ItemList.Circuit_Parts_DiodeSMD.get(64L), 1200, 120);
-        GT_Values.RA.addAssemblerRecipe(new ItemStack[]{GT_OreDictUnificator.get(OrePrefixes.wireFine, Materials.NiobiumTitanium, 16), GT_OreDictUnificator.get(OrePrefixes.dust, Materials.IndiumGalliumPhosphide, 1), GT_Utility.getIntegratedCircuit(3)}, Materials.Polybenzimidazole.getMolten(288L), ItemList.Circuit_Parts_DiodeASMD.get(64L), 600, 480);
+        GT_Values.RA.addAssemblerRecipe(new ItemStack[]{GT_OreDictUnificator.get(OrePrefixes.dust, Materials.IndiumGalliumPhosphide, 2), GT_OreDictUnificator.get(OrePrefixes.bolt, Materials.Iridium, 16), GT_OreDictUnificator.get(OrePrefixes.wireFine, Materials.NiobiumTitanium, 16), GT_OreDictUnificator.get(OrePrefixes.wireFine, Materials.VividAlloy, 16), GT_Utility.getIntegratedCircuit(3)}, Materials.Polybenzimidazole.getMolten(576L), ItemList.Circuit_Parts_DiodeASMD.get(32L), 300, 1920);
```

Comment:
> The original ASMD recipes were FAR worse

4x IndiumGalliumPhosphide + 4 iridium + 4 vivid alloy (what the hell is vivid alloy and where used?) for stack instead of original recipe

>  (apparently they recently were updated to be a stack of 64 instead of their original values?)

with the same material cost for satck

Reply:
It's 2x IndiumGalliumPhsphide, not 4. And yeah, 4 iridium doesn't seem as bad, though at least in my general opinion of doing the Plat Line as it's one of the more tamer rare metals to get. Vivid Alloy is an EiO alloy equivalent to Vibrant Alloy, though it is unused. Sure this particular recipe is considerably more "nerfed" than the previous, however, in earlier versions of the ASMDs, it you required 4x InGaP per 64 ASMD Resistors. This recent patch I've made (still in drafting) has made them just a stack of 64 as a baidaid fix until a solution can arrive later on.

Reply:
1->2 in, 64->32 out means 4x

Reply:
> it you required 4x InGaP per 64 ASMD Resistors

nope
this is diode. not resistors.
and it was 1 for 64
![image](https://user-images.githubusercontent.com/2212585/141438735-2fe2f12c-2f5b-4597-b4ef-189cb80766bf.png)

Reply:
Yeah, I had just realized that I had ended up changing it to 32x instead of 64x